src/RGUnityBots/Assets/Scripts/LegacyInputTests/LegacyMouseHandler.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using UnityEngine;
+
+public class LegacyMouseHandler : MonoBehaviour
```

Comment:
As pointed out below these scripts are in the RGUnityBots test project, not the SDK, so they would not be visible to any projects that import the package. 

I placed these particular scripts in the Scripts directory because they are used as part of the test scene "LegacyInputTestScene". The actual test cases themselves that interact with the scene are indeed in the Tests directory.